Test/Test.hs

```diff
@@ -966,14 +936,82 @@ main = do
   -- insert (table9, table9columndata)
 
   -- Need to run quickcheck after table data has been inserted
-  QuickCheck.run conn
-
-  results <- mapM ($ conn) allTests
-
-  print results
-
-  let passed = and results
-
-  putStrLn (if passed then "All passed" else "Failure")
-  Exit.exitWith (if passed then Exit.ExitSuccess
-                           else Exit.ExitFailure 1)
+  -- QuickCheck.run conn
```

Comment:
yes. I disable it during development because it takes significantly longer to run than the rest of the test suite.